conf/evolutions/default/28.sql

```diff
@@ -0,0 +1,141 @@
+# --- MapRoulette Scheme
+
+# --- !Ups
+
+-- Add Mapped on timestamp
+ALTER TABLE "tasks" ADD COLUMN mapped_on timestamp without time zone DEFAULT NULL;;
```

Comment:
See comments above. I think we should consider a separate table for task reviews.